morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java

```diff
@@ -60,6 +60,14 @@ public class UpgradePathFinder {
 
   private final List<CandidateStep> stepsToApply;
 
+  private final String sourceSchema = "expected schema based on application binaries";
+
+  private final String upgradedSchema = "database schema with upgrades applied";
+
+  private final String reversedSchema = "upgraded database schema with upgrades reversed";
+
+  private final String currentSchema = "current database schema";
+

```

Comment:
`private static final String`

Reply:
Rename `sourceSchema` to `applicationSchema`. Alternatively `targetSchema` could work?

Reply:
Changed to `applicationSchema`